### PR TITLE
Add blog post on backlog cleanup

### PR DESCRIPTION
**Pull request by colebow**
Diving into backlog cleanup & future efforts to keep it small.

### CODE REVIEW

File: _posts/2023-01-10-cleaning-up-the-trino-backlog.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+result should be that nothing slips through the cracks and ends up going months
+without human contact. If an author vanishes or everyone gets too busy to look
+at a pull request again, though, the final stop will ultimately be a stale bot
+which closes pull requests that have gone a few months with no activity.
+
+With all these processes in place, contributors should never feel like their
+efforts are going unnoticed. Submitted code should be reviewed quickly,
+iterated on in a timely manner, and merged without much delay. In situations
+where a pull request is *not* going to be merged, the Trino developer relations
+should be able to chime in quickly to make that clear, saving contributors from
```
Comment (colebow) [inline _posts/2023-01-10-cleaning-up-the-trino-backlog.md:106]:
```suggestion
where a pull request is *not* going to be merged, the Trino developer relations
team should be able to chime in quickly to make that clear, saving contributors from
```

Comment (colebow) [reply _posts/2023-01-10-cleaning-up-the-trino-backlog.md:106]: Just commenting this so I don't forget, I'll tackle it when addressing other feedback.

File: _posts/2023-01-10-cleaning-up-the-trino-backlog.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title:  "Cleaning up the Trino pull request backlog"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+---
+
+At every point in an open source project's lifecycle, it *will* reach a point
```
Comment (mosabua) [inline _posts/2023-01-10-cleaning-up-the-trino-backlog.md:8]:
```suggestion
At some point in the lifecycle of a successful open source project, it *reaches* a point
```

File: _posts/2023-01-10-cleaning-up-the-trino-backlog.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+where the number of incoming pull requests outpace the project's ability to get
+code merged. It happens for a huge variety of reasons, whether that be
```
Comment (mosabua) [inline _posts/2023-01-10-cleaning-up-the-trino-backlog.md:10]: can we avoid "whether that be"

File: _posts/2023-01-10-cleaning-up-the-trino-backlog.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title:  "Cleaning up the Trino pull request backlog"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+---
+
+At every point in an open source project's lifecycle, it *will* reach a point
+where the number of incoming pull requests outpace the project's ability to get
```
Comment (mosabua) [inline _posts/2023-01-10-cleaning-up-the-trino-backlog.md:9]:
```suggestion
where the number of incoming pull requests (PRs) outpace the project's ability to get
```

File: _posts/2023-01-10-cleaning-up-the-trino-backlog.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+code merged. It happens for a huge variety of reasons, whether that be
+developers moving on to other projects before tying up every loose end,
+reviewers who miss a request for review (the GitHub notification system doesn't
```
Comment (mosabua) [inline _posts/2023-01-10-cleaning-up-the-trino-backlog.md:12]: Move the notification thing into a separate sentence.

File: _posts/2023-01-10-cleaning-up-the-trino-backlog.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+do anyone any favors), or because there's a few dozen open PRs which were never
+going to happen and should have been closed two years ago. Having too many open
+PRs can be a problem for a project, because they make it harder to tell what is
```
Comment (mosabua) [inline _posts/2023-01-10-cleaning-up-the-trino-backlog.md:15]: is a problem

File: _posts/2023-01-10-cleaning-up-the-trino-backlog.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title:  "Cleaning up the Trino pull request backlog"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
```
Comment (mosabua) [inline _posts/2023-01-10-cleaning-up-the-trino-backlog.md:5]: Need a highlight image

File: _posts/2023-01-10-cleaning-up-the-trino-backlog.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+To anyone who's managed a repository before, this may seem like more effort than
+necessary. You can add a bot to close anything that's been stale or inactive for
+too long, and problem solved, right? Sure, that does solve the problem, but it
+creates a couple others.
+
+First, and perhaps most importantly: it's not very human. Having a pull request
+that you put time and effort into get shut down by a bot without having another
+person swing by to say hello can be demoralizing, and it builds a negative
+experience that might discourage future contributions to the project. We want
+our contributors to like Trino and to enjoy the process of adding on to it, and
```
Comment (mosabua) [inline _posts/2023-01-10-cleaning-up-the-trino-backlog.md:46]:
```suggestion
our contributors to like the Trino community and to enjoy the process of adding on to it, and
```

File: _posts/2023-01-10-cleaning-up-the-trino-backlog.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+
+The second category is pull requests that get some reviews, but eventually
+stagnate or stop being actively worked on. This happens for a lot of reasons,
+but in all cases, if a pull request goes a few weeks with no activity, the
+developer relations team will be checking in. Our goal will be to figure out the
+proper path forward, whether that's flagging down some reviewers again,
+communicating that the pull request should be closed, or anything else. The end
+result should be that nothing slips through the cracks and ends up going months
+without human contact. If an author vanishes or everyone gets too busy to look
+at a pull request again, though, the final stop will ultimately be a stale bot
```
Comment (mosabua) [inline _posts/2023-01-10-cleaning-up-the-trino-backlog.md:99]: Will it ? Or we just going manual .. might easy enough..

**Review by mosabua — CHANGES_REQUESTED** [review]
A couple of minor things plus

can you add some section titles to break things up?

File: _posts/2023-01-10-cleaning-up-the-trino-backlog.md (hunk 8 above)
Comment (colebow) [reply _posts/2023-01-10-cleaning-up-the-trino-backlog.md:46]: I think this loses meaning.

File: _posts/2023-01-10-cleaning-up-the-trino-backlog.md (hunk 9 above)
Comment (colebow) [reply _posts/2023-01-10-cleaning-up-the-trino-backlog.md:99]: Yeah, we've already got buy-in from maintainers on that.

File: _posts/2023-01-09-cleaning-up-the-trino-backlog.md [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+which closes pull requests that have gone a few months with no activity.
+
+With all these processes in place, contributors should never feel like their
+efforts are going unnoticed. Submitted code should be reviewed quickly,
+iterated on in a timely manner, and merged without much delay. In situations
+where a pull request is *not* going to be merged, the Trino developer relations
+team should be able to chime in quickly to make that clear, saving contributors
+from wasting time and effort on a false impression that their code will be
+landed. And if you have any questions, concerns, or suggestions about all of
+this, don't hesitate to reach out to us directly on the Trino Slack!
```
Comment (bitsondatadev) [inline _posts/2023-01-09-cleaning-up-the-trino-backlog.md:114]:
```suggestion
this, don't hesitate to reach out to us directly on the Trino Slack using `@devrel-team`!
```

**Review by bitsondatadev — APPROVED** [review]
Couple nits and suggestions to add a few lines. If you can get those updated then let's merge!

**Comment by bitsondatadev** [comment]
Looks good thanks dude!

**Pull request merged**